Source/ARM/DigitalTwinsProvisioning-Demo/SmartHotel_Brand 1_Provisioning.yaml

```diff
@@ -0,0 +1,5193 @@
+        sensors:
+        - dataType: Motion
+          type: Classic
+      - name: Light
+        hardwareId: Brand 1_Hotel H 1_610_L
+        sensors:
+        - dataType: Light
+          type: Classic
+    - name: KingRoom 611
+      description: KingRoom 611
+      friendlyName: KingRoom 611
+      type: KingRoom
+      devices:
+      - name: Thermostat
+        hardwareId: Brand 1_Hotel H 1_611_T
+        sensors:
+        - dataType: Temperature
+          type: Classic
+      - name: Motion
+        hardwareId: Brand 1_Hotel H 1_611_M
+        sensors:
+        - dataType: Motion
+          type: Classic
+      - name: Light
+        hardwareId: Brand 1_Hotel H 1_611_L
+        sensors:
+        - dataType: Light
+          type: Classic
+    - name: KingRoom 612
+      description: KingRoom 612
+      friendlyName: KingRoom 612
+      type: KingRoom
+      devices:
+      - name: Thermostat
+        hardwareId: Brand 1_Hotel H 1_612_T
+        sensors:
+        - dataType: Temperature
+          type: Classic
+      - name: Motion
+        hardwareId: Brand 1_Hotel H 1_612_M
+        sensors:
+        - dataType: Motion
+          type: Classic
+      - name: Light
+        hardwareId: Brand 1_Hotel H 1_612_L
+        sensors:
+        - dataType: Light
+          type: Classic
+    - name: KingRoom 613
+      description: KingRoom 613
+      friendlyName: KingRoom 613
+      type: KingRoom
+      devices:
+      - name: Thermostat
+        hardwareId: Brand 1_Hotel H 1_613_T
+        sensors:
+        - dataType: Temperature
+          type: Classic
+      - name: Motion
+        hardwareId: Brand 1_Hotel H 1_613_M
+        sensors:
+        - dataType: Motion
+          type: Classic
+      - name: Light
+        hardwareId: Brand 1_Hotel H 1_613_L
+        sensors:
+        - dataType: Light
+          type: Classic
+    - name: KingRoom 614
+      description: KingRoom 614
+      friendlyName: KingRoom 614
+      type: KingRoom
+      devices:
+      - name: Thermostat
+        hardwareId: Brand 1_Hotel H 1_614_T
+        sensors:
+        - dataType: Temperature
+          type: Classic
+      - name: Motion
+        hardwareId: Brand 1_Hotel H 1_614_M
+        sensors:
+        - dataType: Motion
+          type: Classic
+      - name: Light
+        hardwareId: Brand 1_Hotel H 1_614_L
+        sensors:
+        - dataType: Light
+          type: Classic
+    - name: Suite 615
+      description: Suite 615
+      friendlyName: Suite 615
+      type: Suite
+      devices:
+      - name: Thermostat
+        hardwareId: Brand 1_Hotel H 1_615_T
+        sensors:
+        - dataType: Temperature
+          type: Classic
+      - name: Motion
+        hardwareId: Brand 1_Hotel H 1_615_M
+        sensors:
+        - dataType: Motion
+          type: Classic
+      - name: Light
+        hardwareId: Brand 1_Hotel H 1_615_L
+        sensors:
+        - dataType: Light
+          type: Classic
+    - name: Suite 616
+      d
```

Comment:
type should be **Floor**
subtype should be **VIP**

the SpaceDescription.cs will need to be updated to support this, but the DigitalTwins.Space has the SubType property on it already

Reply:
Type is now Floor, but in order for the subtype to work Digital Twins requires it to be VIPFloor